internal/output/table.go

```diff
@@ -363,7 +363,7 @@ func filterZeroValComponents(costComponents []CostComponent, resourceName string
 	var filteredComponents []CostComponent
 	for _, c := range costComponents {
 		if c.MonthlyQuantity != nil && c.MonthlyQuantity.IsZero() {
-			log.Info().Msgf("Hiding cost with no usage: %s '%s'", resourceName, c.Name)
+			log.Debug().Msgf("Hiding cost with no usage: %s '%s'", resourceName, c.Name)
```

Comment:
There's two other places we log this still with info level:
* https://github.com/infracost/infracost/blob/master/internal/output/table.go#L223
* https://github.com/infracost/infracost/blob/master/internal/output/html.go#L41